libgdx/vector-pinball/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed May 24 11:47:06 MSK 2017
+#Fri Jun 30 18:38:43 MSK 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.4.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.0-bin.zip
```

Comment:
Gradle wrapper is a bit problematic in this. Because `gradle-wrapper.jar` is a binary file and the actual contents depend on the version so I have to download it and compare containing files with other gradle file.

That's why I cannot validate it with just my mobile phone and need a computer to do this.